db/migrate/20210701000410_truncate_old_participant_declaration_data.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class TruncateOldParticipantDeclarationData < ActiveRecord::Migration[6.1]
+  def up
+    ParticipantDeclaration.delete_all
+  end
+
+  def down
+    ParticipantDeclaration.delete_all
+    ProfileDeclaration.delete_all
+  end
```

Comment:
Not sure about this one... Why does it need to be a migration, and why does it need to be reversible?

Reply:
It didn't make much sense to keep records that aren't used for anything apart from the ones on the sandbox for testing the API endpoint, so rather than a migration that converted them to user_ids from early_career_teacher_profile_ids, just dropping them felt more like a pragmatic solution. 
The down can probably be dropped, but down then up could potentially have extra (useless) records in the ProfileDeclaration so either the ProfileDeclarations should be dropped on down, or added to the delete_all on up (or even both).

Reply:
I would expect this to happen on the sandbox console. I think migrations are for schema changes, and when it is correct to change data in them, that should be focussed on production

Reply:
In principle, I agree, but this is tied up to a migration, so we'd have to perform it or the migrations will fail.

https://codingitwrong.com/2016/07/29/how-to-seed-and-migrate-rails-data.html

Particularly,

2. Transform Data Outside of Migrations When Possible
<removed for brevity>
**This isn’t always possible, however. The main time it’s not possible is when data needs to be transformed in order to do a schema migration. For example, if you’re changing what table a column is stored in, you will want to add the new column, transfer the data over, and then drop the old column. This can’t be done easily outside of schema migrations. And in this case, the data transformation is closely tied to the schema change, so keeping it in a migration makes sense.**

Reply:
Which is what it was originally going to do. I'm not wedded to doing this in the migration though. If we're ok with a manual console change and control when the deploy happens to ensure that it's timely enough, I'm ok with dropping it from this PR.

Reply:
Dropped the change. We just need to make sure that we do this manually.